portfolio/styles/LayoutLMInferenceCarousel.module.css

```
@@ -0,0 +1,224 @@
+
+.wordControls {
+  display: flex;
+  gap: 0.5rem;
+}
+
+.controlButton {
+  padding: 0.5rem 0.75rem;
+  background: var(--button-background);
+  color: var(--button-text);
+  border: 1px solid var(--text-color);
+  border-opacity: 0.2;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)